presto-parser/src/main/java/com/facebook/presto/sql/tree/SimpleGroupBy.java

```diff
@@ -42,7 +42,7 @@ public SimpleGroupBy(NodeLocation location, List<Expression> simpleGroupByExpres
     private SimpleGroupBy(Optional<NodeLocation> location, List<Expression> simpleGroupByExpressions)
     {
         super(location);
-        this.columns = requireNonNull(simpleGroupByExpressions);
+        this.columns = ImmutableList.copyOf(requireNonNull(simpleGroupByExpressions, "simpleGroupByExpressions is null"));
```

Comment:
In this case, this can be made easier to read by renaming the parameter from `simpleGroupByExpressions` to `columns`.